src/lib.rs

```diff
@@ -2764,12 +2766,10 @@ pub unsafe trait FromBytes: FromZeros {
     /// Interprets the prefix of the given `bytes` as a `&[Self]` with length
     /// equal to `count` without copying.
     ///
-    /// This method verifies that `bytes.len() >= size_of::<T>() * count` and
-    /// that `bytes` is aligned to `align_of::<T>()`. It reinterprets the first
-    /// `size_of::<T>() * count` bytes from `bytes` to construct a `&[Self]`,
-    /// and returns the remaining bytes to the caller. It also ensures that
-    /// `sizeof::<T>() * count` does not overflow a `usize`. If any of the
-    /// length, alignment, or overflow checks fail, it returns `Err`.
+    /// This method attempts to return a reference to the prefix of `bytes`
+    /// reinterpreted as a `Self` with `count` trailing elements, and reference
+    /// to the preceeding bytes. If there are insufficient bytes, or if `bytes`
```

Comment:
```suggestion
    /// to the remaining bytes. If there are insufficient bytes, or if `bytes`
```

Or something else, but "preceding" only works if this were a suffix cast.

Reply:
Done